.github/config/linux.json

```diff
@@ -7,9 +7,9 @@
     "linux-x64x11",
     "linux-x64x11-egl",
     "linux-x64x11-skia",
-    "linux-x64x11-sbversion-13",
     "linux-x64x11-sbversion-14",
-    "linux-x64x11-sbversion-15"
+    "linux-x64x11-sbversion-15",
```

Comment:
doesn't Raspi-2 also need updates here? We build it at least for one sbversion

Reply:
Done